GoogleForCreators/web-stories-wp
# Media: Enable Hotlinking by default

## Context



## Summary



Enables media hotlinking (and  the CORS proxy) feature by default.

## Relevant Technical Choices



## To-do



## User-facing changes



![Screenshot 2021-11-24 at 13 32 39](https://user-images.githubusercontent.com/841956/143239046-25a0b137-9416-49dc-8d06-f00f168ced63.png)

## Testing Instructions






This PR can be tested by following these steps:

1. Use the Hotlinking button in the Media tab (next to the Upload button) to insert different images and videos into the Page
2. For example try linking media from the site itself; from Unsplash; from wp.stories.google
3. Verify that a new media element is created to the page, and while the media is being inserted, the Insert button is disabled.
4. Verify that inserting incorrect URLs (such as faulty URLs, links that point to a file that's not image/video, etc.) display an error message and are not inserted.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9466
